src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactElasticWellbore/Example.rst

```diff
@@ -0,0 +1,159 @@
+.. _AdvancedExampleCasedElasticWellbore_ImperfectInterfaces:
+
+
+###############################################
+Cased Elastic Wellbore with Impefect Interfaces
+###############################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example uses the ``LagrangianContact`` solver to handle a cased wellbore problem with imperfect contact interfaces. The completed wellbore is composed of a steel casing, a cement sheath, and rock formation. All the three materials are assumed to exhibit isotropic linear elastic behavior. The contact surfaces between these materials follow the Lagrangian contact law. 
+
+.. _problemSketchCasedElasticWellboreInterfacesFig:
+.. figure:: sketch.png
+   :align: center
+   :width: 600
+   :figclass: align-center
+
+   A cased wellbore with imperfect casing-cement and cement-rock interfaces 
+
+Under a radial compression loading, the imperfect contact intefaces behave like the perfect ones (see :ref:`AdvancedExampleCasedElasticWellbore`). When a radial tension loading on the inner face of the wellbore, the casing debonds from the cement layer. Analytical results of the radial displacement :math:`u_{r}` in the casing is expressed as `(Hervé and Zaoui, 1995) <https://link.springer.com/chapter/10.1007%2F978-94-015-8494-4_55>`__ :
+
+.. math::
+   u_{r} = Ar - \frac{B}{r}
+
+where :math:`r` is the radial coordinate, :math:`A` and :math:`B` are constants that are obtained by solving the boundary conditions, as detailed in the post-processing script. The outer face of the casing as well as the inner face of the cement layer are free of stress because of the debonding at the casing-cement interface. Therefore, the displacement jump at the cement-rock interface is nil, and the displacement jump accross the casing-cement interface is equal to :math:`u_r(r=r_{out_casing})`, where :math:`r_{out_casing}` is the outer raidus of the casing. 
```

Comment:
```suggestion
where :math:`r` is the radial coordinate, :math:`A` and :math:`B` are constants that are obtained by solving the boundary conditions, as detailed in the post-processing script. The outer face of the casing as well as the inner face of the cement layer are free of stress because of the debonding at the casing-cement interface. Therefore, the displacement jump at the cement-rock interface is null, and the displacement jump across the casing-cement interface is equal to :math:`u_r(r=r_{out_casing})`, where :math:`r_{out_casing}` is the outer radius of the casing. 
```

Reply:
done